### DISCUSSION TITLE
Package.json  error

### DISCUSSION THREAD

**Discussion by SrushtiPatiL25** [smaranjitghose/awesome-portfolio-websites#1138] [Q&A]
when I npm start the project its giving me error
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'F:\awesome-portfolio-websites\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
please help

**Comment by anushbhatia** [comment]
In case, if you are using code on the main branch it isn't react code. Clone the code present on react branch.

Then run 
- npm install
- npm start
